README.md

```diff
@@ -12,8 +12,7 @@ The purpose of this challenge is to construct complicated queries, learn about j
 - Install dependencies with `$ npm install`
 - Create an app on Heroku `heroku create app-name-here --region eu` (If you haven't yet installed Heroku CLI see [here](https://devcenter.heroku.com/articles/heroku-cli)
 - Push to Heroku `git push heroku master`
-- Create a new database on Heroku: `heroku addons:create heroku-postgresql:hobby-dev`
-(You can also create the database with an alias `heroku addons:create heroku-postgresql:hobby-dev --as USERS_DB`)
+- Create a new database on Heroku: `heroku addons:create heroku-postgresql:hobby-dev --as USERS_DB`
```

Comment:
Is it worth mixing in the old description for this step eg:

Create a new database on Heroku with an alias
_OR_
Create a new database on Heroku with the alias USERS_DB

Reply:
doesn't that re-introduce the original problem?